README.md

```diff
@@ -34,6 +34,7 @@ The following parts of [OpenID Connect Core 1.0](http://openid.net/specs/openid-
 - [Requesting Claims using Scope Values](http://openid.net/specs/openid-connect-core-1_0.html#ScopeClaims)
 - [UserInfo Endpoint](http://openid.net/specs/openid-connect-core-1_0.html#UserInfo)
 - [Normal Claims](http://openid.net/specs/openid-connect-core-1_0.html#NormalClaims)
+- (From doorkeeper v5.5.0) [OAuth 2.0 Form Post Response Mode](https://openid.net/specs/oauth-v2-form-post-response-mode-1_0.html)
```

Comment:
**nitpick:** Not sure if we should specify the Doorkeeper version here in the README, as we don't do it for other features either :wink: 

But we should mention it in the CHANGELOG, something like `This gem now requires Doorkeeper 5.5`.

Reply:
It fixed! https://github.com/doorkeeper-gem/doorkeeper-openid_connect/pull/138/commits/99ef25ed9b6949c64d5aff08e7b29e6349cb393e